### ISSUE TITLE
rawspec Makefile doesn't include newer GPU SM architectures

### ISSUE THREAD

**Issue by khouston22** [UCBerkeleySETI/rawspec#73]
rawspec as currently configured won't run for newer GPU compute architectures because the appropriate PTX code won't be generated. 

Makefile line 64 is this: 
SMS ?= 35 50 52 60 61 70 75
which implies that code will be generated for sm_35, sm_50, ... sm_75 architectures

For nvcc release 12.3, "nvcc --list-gpu-code" lists the following supported architectures:
sm_50
sm_52
sm_53
sm_60
sm_61
sm_62
sm_70
sm_72
sm_75
sm_80
sm_86
sm_87
sm_89
sm_90
which implies line 64 should be:
SMS ?= 35 50 52 53 60 61 62 70 72 75 80 86 87 89 90
Perhaps in practice not all values need support.  I personally needed sm_86 for my RTX 3070 setup.

**Comment by david-macmahon** [comment]
Yeah, it probably is time to update the default list, but in the meantime you can specify your own `SMS` list on the `make` command line:

```
make SMS="60 61 70 75 86"
```

(If the list has multiple entries, be sure to use quotes as shown above so that the shell treats it as a single string)

**Comment by khouston22** [comment]
Of course, I’ve already done this.  Perhaps a note in the readme would be
useful describing the workaround you describe here.
